src/editors/data/services/cms/api.js

```diff
@@ -180,6 +180,29 @@ export const apiMethods = {
           license: module.processLicense(content.licenseType, content.licenseDetails),
         },
       };
+    } else if (blockType ==='library_content') {
+      response = {
+        category: blockType,
+        courseKey: learningContextId,
+        display_name: title, //"course-v1:edx+123+test"
+        // has_children: true,
+        highlights: [],
+        highlights_doc_url: "",
+        highlights_enabled: false,
+        highlights_enabled_for_messaging: false,
+        highlights_preview_only: true,
+        id: blockId, //"block-v1:edx+123+test+type@library_content+block@880758e62c2542d5b45c944360dfa799"
+        metadata: {
+          allow_resetting_children: content.showReset,
+          candidates: content.candidates,
+          capa_type: 'any',
+          count: content.count,
+          //manual
+          //shuffle
```

Comment:
We'll want to uncomment these.
```suggestion
          manual
          shuffle
```